frontend/src/components/BigPopup/CommentSection/CommentSection.tsx

```diff
@@ -26,12 +32,30 @@ const CommentSection = () => {
 
   const topComments = comments.filter(c => !c.parentComment);
 
+  const [topCommentsOnPageIndex, setTopCommentsOnPageIndex] =
+    useState(commentStepCount);
+
   useEffect(() => {
     if (locationId) {
-      fetchComments(locationId);
+      fetchComments(locationId, undefined, topCommentsOnPageIndex);
     }
   }, [locationId]);
 
+  useEffect(() => {
+    const addMoreComment = () =>
+      setTopCommentsOnPageIndex(prevState => prevState + commentStepCount);
+
+    const timerId = setTimeout(() => addMoreComment(), 2000);
+
+    return () => {
+      clearTimeout(timerId);
+    };
```

Comment:
remove this

Reply:
I change timeout to 500 (0,5 s ) , I Like when i see title about downloading new comment
![image](https://user-images.githubusercontent.com/37217530/179173856-065e1fc8-61e2-4cb2-9cc1-852136e3abac.png)

If I remove timeout  - I cant see it